### PR TITLE
Translation from knagasaki/TEI

### PR DESCRIPTION
**Pull request by knagasaki**

### CODE REVIEW

**Comment by knagasaki** [comment]
Revised JPN translation.

**Comment by knagasaki** [comment]
Added and revised JPN translations

**Comment by knagasaki** [comment]
Add some Japanese translation.

**Comment by knagasaki** [comment]
Added Japanese translation for corresp-related elements.

File: P5/Source/Specs/accMat.xml [hunk 1]
```diff
@@ -18,11 +18,7 @@ described, such as non-contemporaneous documents or fragments bound in
 with it at some earlier historical period.</desc>
   <desc versionDate="2007-12-20" xml:lang="ko">기술하고 있는 원고와 밀접히 관련된 중요한 부가적 자료를 세밀하게 기술한다. 예를 들어, 어떤 이전 시기에 원고와 함께 제본된 비동시대의 문서  또는 문서 일부.</desc>
   <desc versionDate="2007-05-02" xml:lang="zh-TW">包含可能與該手稿關係密切的重要附加資料之細節，例如非同時期的文件或是在早期與手稿裝訂一起的片段。</desc>
-  <desc versionDate="2008-04-05" xml:lang="ja">当該手書き資料と関連する情
-  報の詳細を記述するための要素．例えば，当該手書き資料が書かれた時代と
-  は異なる時期の文書や断片についてについての詳細を示す．例えば，当該手
-  書き資料が作成された以前に書かれた文書やその断片であったりするものに
-  ついて．
+  <desc versionDate="2008-04-05" xml:lang="ja">当該手書き資料と密接に関連していると考えられる重要な追加資料，例えば、時代を異にする文書，歴史的に早い時期に手書き資料と合わされた断片などの詳細を示す．
```
Comment (peterstadler) [inline P5/Source/Specs/accMat.xml]: `@versionDate` should be updated

File: P5/Source/Specs/additions.xml [hunk 2]
```diff
@@ -14,8 +14,7 @@ $Id$
 within a manuscript or other object, such as marginalia or other annotations.</desc>
   <desc versionDate="2007-12-20" xml:lang="ko">방주 또는 기타 주석과 같이 원고 내의 중요한 부가정보의 기술을 포함한다.</desc>
   <desc versionDate="2007-05-02" xml:lang="zh-TW">包含手稿中任何重要增添文字的描述，例如旁註或其他註釋。</desc>
-  <desc versionDate="2008-04-05" xml:lang="ja">当該手書き資料中にある，重要な付加物を示す．例えば，欄外の書き込みや
-  注釈など．</desc>
+  <desc versionDate="2008-04-05" xml:lang="ja">当該手書き資料中にある，重要な付加物を示す．例えば，欄外の書き込みや注釈など．</desc>
```
Comment (peterstadler) [inline P5/Source/Specs/additions.xml]: `@versionDate` should be updated

File: P5/Source/Specs/affiliation.xml [hunk 3]
```diff
@@ -14,7 +14,7 @@
         organization, for example an employer or sponsor.</desc>
   <desc versionDate="2007-12-20" xml:lang="ko">고용주 또는 후원자와 같이 개인의 현재 또는 과거 소속 조직에 대한 비공식적 기술을 포함한다.</desc>
   <desc versionDate="2007-05-02" xml:lang="zh-TW">包含非正式性的描述，關於個人現在或過去隸屬的團體，例如雇主或贊助者。</desc>
-  <desc versionDate="2008-04-05" xml:lang="ja">人物が所属している，またはしていた団体に関する情報を示す．例えば，雇 い主や出資者など．</desc>
+  <desc versionDate="2008-04-05" xml:lang="ja">人物が所属している，またはしていた団体に関する情報を示す．例えば，雇い主や出資者など．</desc>
```
Comment (peterstadler) [inline P5/Source/Specs/affiliation.xml]: `@versionDate` should be updated

File: P5/Source/Specs/am.xml [hunk 4]
```diff
@@ -16,10 +16,7 @@ $Id$
   the abbreviation.</desc>
   <desc versionDate="2007-12-20" xml:lang="ko">축약의 확장형에서 생략되거나 대체된 축약형으로, 제시된 문자열 또는 기호열을 포함한다.</desc>
   <desc versionDate="2008-04-06" xml:lang="es">contiene una secuencia de letras o símbolos  presentes en una abreviatura que han sido omitidos o substituidos en la forma extendida de la abreviatura.</desc>
-  <desc versionDate="2008-04-05" xml:lang="ja">
-  当該省略形が，原形を省略またはそれに代わったことを示す文字列または記
-  号列．
-</desc>
+  <desc versionDate="2008-04-05" xml:lang="ja">当該省略形が，原形を省略またはそれに代わったことを示す文字列または記号列．</desc>
```
Comment (peterstadler) [inline P5/Source/Specs/am.xml]: `@versionDate` should be updated

File: P5/Source/Specs/att.measurement.xml [hunk 5]
```diff
@@ -46,7 +46,8 @@ $Id$
           <gloss versionDate="2007-05-02" xml:lang="zh-TW">公尺</gloss>
           <gloss versionDate="2007-06-12" xml:lang="fr">mètre</gloss>
           <gloss versionDate="2007-11-06" xml:lang="it">metro</gloss>
-          <gloss versionDate="2007-05-04" xml:lang="es">metro</gloss>
+          <gloss versionDate="2022-05-12" xml:lang="ja">metro</gloss>
+	  <gloss versionDate="2007-11-06" xml:lang="it">メートル</gloss>
```
Comment (peterstadler) [inline P5/Source/Specs/att.measurement.xml]: probably a copy paste error here?! duplicate Italian gloss and the Japanese gloss in Italian?

**Review by peterstadler — CHANGES_REQUESTED** [review]
Many thanks for this PR! While I'm unable to review the actual content, I found some technical issues that should be addressed.

File: P5/Source/Specs/accMat.xml (hunk 1 above)
Comment (knagasaki) [reply P5/Source/Specs/accMat.xml]: Dear Peter, thank you for your cheking. I've updated the date.

File: P5/Source/Specs/additions.xml (hunk 2 above)
Comment (knagasaki) [reply P5/Source/Specs/additions.xml]: Dear Peter, thank you for your cheking. I've updated the date.

File: P5/Source/Specs/am.xml (hunk 4 above)
Comment (knagasaki) [reply P5/Source/Specs/am.xml]: Dear Peter, thank you for your cheking. I've updated the date.

File: P5/Source/Specs/att.measurement.xml (hunk 5 above)
Comment (knagasaki) [reply P5/Source/Specs/att.measurement.xml]: Dear Peter, thank you for your cheking. I've deleted the duplication.

**Comment by knagasaki** [comment]
I've revised the conflict. I'd be glad if it would be merged.

File: P5/Source/Specs/att.measurement.xml [hunk 6]
```diff
@@ -46,7 +46,7 @@ $Id$
           <gloss versionDate="2007-05-02" xml:lang="zh-TW">公尺</gloss>
           <gloss versionDate="2007-06-12" xml:lang="fr">mètre</gloss>
           <gloss versionDate="2007-11-06" xml:lang="it">metro</gloss>
-          <gloss versionDate="2007-05-04" xml:lang="es">metro</gloss>
+          <gloss versionDate="2022-05-12" xml:lang="ja">metro</gloss>
```
Comment (peterstadler) [inline P5/Source/Specs/att.measurement.xml:50]: The 'es' should not be overwritten here

**Review by peterstadler — CHANGES_REQUESTED** [review]

File: P5/Source/Specs/att.measurement.xml (hunk 6 above)
Comment (knagasaki) [reply P5/Source/Specs/att.measurement.xml:50]: Thank you very much for your pointing it out. I've solved it.

Comment (peterstadler) [reply P5/Source/Specs/att.measurement.xml:50]: Wow, that was fast! Thank you @knagasaki !

File: P5/Source/Specs/att.cReferencing.xml [hunk 7]
```diff
@@ -16,13 +16,13 @@
       <gloss versionDate="2008-04-06" xml:lang="es">referencia canónica</gloss>
       <gloss versionDate="2008-03-30" xml:lang="fr">référence canonique</gloss>
       <gloss versionDate="2007-11-06" xml:lang="it">riferimento canonico</gloss>
-      <gloss xml:lang="ja" versionDate="2018-12-31">正規化参照</gloss>
+      <gloss xml:lang="ja" versionDate="2018-12-31">基準的形式の参照</gloss>
```
Comment (peterstadler) [inline P5/Source/Specs/att.cReferencing.xml]: The `@versionDate` should probably be updated as well

Comment (knagasaki) [reply P5/Source/Specs/att.cReferencing.xml]: I'm sorry for the mistake. I've solved it.

**Review by peterstadler — APPROVED** [review]
Thank you very much!!

**Comment by peterstadler** [comment]
This looks technically good to me, @martindholmes could you have a second look and eventually merge?

**Comment by martindholmes** [comment]
@knagasaki I have a question about punctuation. I notice that the recent translations use western punctuation marks (, for comma, . for period) whereas existing translations in the Guidelines use Japanese punctuation marks (、U+3001 for comma, 。U+3002 for period). I think we should be consistent throughout the Guidelines. Which do you think is best? I can easily make them all consistent after merging your current pull requests.

**Comment by knagasaki** [comment]
Thank you for checking that. In the current guidelines, "． U+FF0E" and "， U+FF0C" are used instead of the ASCII code ". U+002E" and ", U+ 002C" for Japanese explanations. Both are western styles, but the former set of character codes is used in Japanese horizontal writings. So, please use them as it is.

**Comment by martindholmes** [comment]
Thank you @knagasaki. That means I will have to convert all the existing punctuation marks in the Guidelines files to use these.

**Comment by knagasaki** [comment]
I apologize if I've misunderstood the point of your suggestion, the current guidelines use the "，(U+FF0C) ．(U+FF0E)" like below, so we followed the style. 

- [desc](https://tei-c.org/release/doc/tei-p5-doc/ja/html/ref-desc.html) 当該要素，属性，属性値を採用した目的の簡単な解説を示す．
- [mapping](https://tei-c.org/release/doc/tei-p5-doc/ja/html/ref-mapping.html) (character mapping) 属性typeで示される，親文字またはグリフと関連する，ひとつ 以上の文字を示す．

**Comment by martindholmes** [comment]
@knagasaki The current guidelines have a mixture of both types of punctuation, as you can see here for example:

https://tei-c.org/release/doc/tei-p5-doc/ja/html/ref-att.combinable.html

I think we should be consistent, but I don't think it matters which style we choose. I'm not used to seeing western punctuation marks used in Japanese, but I mostly read novels which are printed vertically, so I looked at some online sources like https://mainichi.jp/ and https://www.yomiuri.co.jp/, and I see Japanese-style punctuation there in horizontal text. What do you think?

**Pull request merged**